src/main/java/util/Parser.java

```diff
@@ -0,0 +1,39 @@
+package util;
+
+import java.util.HashMap;
+import java.util.InputMismatchException;
+
+public interface Parser {
+    static String getCommand(String input) {
+        return input.split(" ", 2)[0];
+    }
+
+    static HashMap<String, String> getArgMap(String input) throws InputMismatchException {
+        HashMap<String, String> argMap = new HashMap<>();
+
+        if (input.split(" ").length < 2) {
+            return argMap;
+        }
+
+        String[] argArr = input.split("/");
+        for (int i = 0; i < argArr.length; i++) {
+            String flag;
+            String param;
```

Comment:
How about using the pair words key and value, since they will be used in a `HashMap`